app/src/components/grids/searchCardsGrid.tsx

```diff
@@ -12,8 +12,11 @@ const SearchCardsGrid = ({ results, keywords }) => {
   const cardRef = useRef<HTMLDivElement | null>(null);
   return (
     <SimpleGrid columns={1} gap="grid.l">
-      {results?.map((result: SearchCardType, index: number) => {
-        const searchResult = new SearchCardModel(result);
+      {results.results?.map((result: SearchCardType, index: number) => {
```

Comment:
somewhat random: you're not using `useTooltipOffset` or ``useCardImageHeight` or `cardRef` in this file.